python/ray/dag/compiled_dag_node.py

```diff
@@ -539,6 +540,10 @@ def __init__(
                 executions is beyond the DAG capacity, the new execution would
                 be blocked in the first place; therefore, this limit is only
                 enforced when it is smaller than the DAG capacity.
+            max_in_flight_requests: The maximum number of in-flight requests that
```

Comment:
`max_in_flight_executions` is more accurate? We have `execute()` calls.

Reply:
Changed it to max_inflight_executions